templates/template.md.ejs

```diff
@@ -43,6 +43,24 @@ browser.<?= docfile.filename.slice(0,-3) ?>(<?= command.paramStr ?>);
 <? }); ?>
 <? } ?>
 
+<? if (command.returnTags.length) { ?>
+### Returns
+
+| Type | Details |
+| ---- | ------- |
+<? command.returnTags.forEach(function(returnTag) { ?>| <?= returnTag.joinedTypes.replace('=','').replace(/[|\/]/g,'/') ?> | <?= returnTag.description ?> |
```

Comment:
I would prefer simple lists here, like:

### Returns
- __\<String\>__ some description about the property
- __\<Object\>__ an object that could look like:
    ```js
    {
        prop: "somevalue",
        prop2: true
    }
    ```

### Throws
- __RuntimeError__ the condition when it throws